Cargo.toml

```diff
@@ -13,3 +13,16 @@ resolver = "2"
 lto = true
 opt-level = 3
 strip = true
+
+[workspace.dependencies]
+nostr-sdk = { version = "0.34", features = [] }
+cdk = "0.4.0"
+anyhow = "1"
+log = "0.4"
+dotenv = "0.15"
+tokio = "1.38"
+serde = "1"
```

Comment:
`serde_json` depends on `serde` , thus it would be better not to specify `serde` on its own and just get it as a transitive dependency.

Reply:
You mean like https://github.com/f321x/cashu-escrow-kit/pull/29/commits/f2e7dcebdd650b4d559fdaa4384a7794b5ea2d02 so it doesn't pull two separate serde versions?
If we don't specify serde at all compiling fails as serde_json does not export serde traits like Deserialize.

Reply:
You are right, I though I was able to omit the serde dependency in the past, but obviously I'm wrong.